src/basic/missing_syscall.h

```diff
@@ -154,6 +154,12 @@ static inline pid_t raw_getpid(void) {
 
 /* ======================================================================= */
 
+static inline pid_t raw_gettid(void) {
+        return (pid_t) syscall(__NR_gettid);
+}
```

Comment:
gettid() is not exposed by glibc anyway, we already define our own in missing_syscall.h, i.e. you can just use gettid()